src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -2060,22 +2059,19 @@ private AssemblySymbol GetForwardedToAssembly(string fullName, int arity, Diagno
             return null;
         }
 
-        internal static void CheckFeatureAvailability(Location location, MessageID feature, DiagnosticBag diagnostics)
+        internal static void CheckFeatureAvailability(SyntaxNode syntax, MessageID feature, DiagnosticBag diagnostics, Location locationOpt = null)
```

Comment:
>Location locationOpt = null [](start = 119, length = 27)

It doesn't look like we are ever taking advantage of this parameter. Consider removing.